api_docs/changelog.md

```diff
@@ -20,6 +20,14 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 272**
+
+* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events),
+  [`GET /streams`](/api/get-streams),
+  [`GET /streams/{stream_id}`](/api/get-stream-by-id): Added a new
+  field `is_recently_active` to stream, which can used to determine if a
+  message has been sent to the stream in the last 180 days.
```

Comment:
The documentation on the field itself should have the full details (all the details that we want to document at all), since that's the authoritative place people look in order to understand the current API. This changelog, on the other hand, can have fewer details since it links there; it just needs enough to give the reader an idea of whether this is a change they're interested in, when they're scanning through what's changed between two versions.

So in particular the detail "in the last 180 days", if we document it at all, should be on the field's docs. But I think we can leave it out from the changelog entry.

Reply:
Makes sense. Updated!